pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopicPolicies.java

```diff
@@ -466,11 +466,16 @@ private class SetRetention extends CliCommand {
         private List<String> params;
 
         @Parameter(names = { "--time",
-                "-t" }, description = "Retention time in minutes (or minutes, hours,days,weeks eg: 100m, 3h, 2d, 5w). "
-                + "0 means no retention and -1 means infinite time retention", required = true)
+                "-t" }, description = "Retention time with optional time unit suffix "
+                + "eg: 100m, 3h, 2d, 5w. "
+                + "If no time unit is specified the default is seconds, eg: "
+                + "-t 120 will set retention to 2 minutes. "
```

Comment:
```suggestion
                + "-t 120 sets retention to 2 minutes. "
```
Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true.
https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit?pli=1#bookmark=id.e8uqh1awkcnp

Reply:
Updated.  Thanks